### PR TITLE
#45 and #34 Fixed file size issue, page doesn't break now.

### PR DESCRIPTION
**Pull request by br0wncrust4cean**
- File size being too big broke the page, just added in validation so we don't have problems.

### CODE REVIEW

File: Project/src/main/resources/static/js/upload.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+function checkFileSize(event) {
+    var tenMegabytes = 10000000; //10 MB in bytes
+    var fileElement = document.getElementById("fileLoad");
+    var fileArr = fileElement.files;
+
+    if (!fileArr || !fileArr[0]) {
+        alert("No file loaded");
```
Comment (DanGravel) [inline Project/src/main/resources/static/js/upload.js:7]: not sure an alert is the best

File: Project/src/main/resources/static/js/upload.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+function checkFileSize(event) {
+    var tenMegabytes = 10000000; //10 MB in bytes
```
Comment (Taz-Z) [inline Project/src/main/resources/static/js/upload.js]: Is there a JS equivalent of static vars? If so pull this up

Comment (Taz-Z) [reply Project/src/main/resources/static/js/upload.js]: And call it fileSizeLimit  or something of that nature if you do pull it up

**Review by Taz-Z — CHANGES_REQUESTED** [review]

File: Project/src/main/resources/static/js/upload.js (hunk 1 above)
Comment (br0wncrust4cean) [reply Project/src/main/resources/static/js/upload.js:7]: What do you suggest then? This literally just makes a popup that there's no file and then they click ok and then they can add in the file.

Comment (DanGravel) [reply Project/src/main/resources/static/js/upload.js:7]: I mean you can do it like that but i think having the alert on the page is better. @Taz-Z can you weigh in.

Comment (br0wncrust4cean) [reply Project/src/main/resources/static/js/upload.js:7]: I think that's fine if you want to do it on the page, but can we wait until we integrate bootstrap? I think errors are more graceful on there.

Comment (Taz-Z) [reply Project/src/main/resources/static/js/upload.js:7]: I say keep it as is for now - wait till bootstrap integration and think of a "nicer" sln

**Review by Taz-Z — APPROVED** [review]

**Review by DanGravel — APPROVED** [review]

**Pull request merged**